src/prop-converter.js

```diff
@@ -28,6 +28,10 @@ import {
 } from './enter-leave-presets';
 import { isElementAnSFC, omit } from './helpers';
 
+// Define process global in case the consumer hasn't defined process.env.NODE_ENV
+if (typeof process === 'undefined') {
+  var process = { env: {} };  // Must use var to escape block scoping
+}
```

Comment:
The reason I used the local variable instead of setting the global is because I wanted to keep the module self-contained.  While I don't foresee any bugs around this, people do weird things in their code and I wanted to avoid modifying the global scope if at all possible.